itou/users/management/commands/pe_certify_users.py

```diff
@@ -0,0 +1,113 @@
+"""
+This command uses the PE API to try and certify job seeker profiles against their
+first name, last name, birthdate and NIR, eventually swapping first and last names
+if needed.
+"""
+
+import logging
+
+import tenacity
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from django.utils import timezone
+
+from itou.users.enums import UserKind
+from itou.users.models import JobSeekerProfile, User
+from itou.utils.apis import pole_emploi_api_client
+from itou.utils.apis.pole_emploi import (
+    PoleEmploiAPIBadResponse,
+    PoleEmploiAPIException,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("--wet-run", action="store_true", dest="wet_run")
+        # default chunk size is chosen to be 200 so that the cron lasts about 3 minutes.
+        parser.add_argument("--chunk-size", action="store", dest="chunk_size", default=200, type=int)
+
+    def handle(self, wet_run, chunk_size, **options):
+        pe_client = pole_emploi_api_client()
+
+        # only retry the PoleEmploiRateLimitException
+        @tenacity.retry(stop=tenacity.stop_after_attempt(10), wait=tenacity.wait_fixed(1))
```

Comment:
A priori _tenacity_ permet de [limiter les retry à certaines exceptions seulement](https://tenacity.readthedocs.io/en/latest/#whether-to-retry).

Reply:
Oui, mais je veux de toutes façons contenir le `try...except` et retourner l'objet exception car ça me permet un code moins ... surindenté derrière. 

Mon commentaire est là pour expliquer l'intention générale.

Reply:
En effet... je vais filtrer sur l'exception.

Reply:
done